oxidation/libparsec/crates/tests_utils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
```

Comment:
What is the need for this crate ? can't we just leave this inside `tests_fixtures` ? (I'd rather avoid adding too much crates if possible)

(if moving this in `tests_fixtures` is not possible, why not moving it in `types`'s `fixtures.rs` ?)

Reply:
It was required before we update the `regex` test

Reply:
This crate was removed